test/utils/objects/pod.go

```diff
@@ -20,13 +20,14 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/openshift-kni/numaresources-operator/test/utils/images"
```

Comment:
the conventional import grouping is
```
stdlib

3rd parties (most known, usually kube)

3rd parties (less known, usually github packages)

local
```
so please move this import in its own group at the bottom of the `import` block

Reply:
thank you, I'll note this for next time. updated